.travis.yml

```diff
@@ -1,27 +1,29 @@
 language: rust
 sudo: false
-matrix:
-  include:
-    - os: osx
+rust:
+  - stable
+  - beta
+  - nightly
+os:
+  - linux
+  - osx
```

Comment:
I'd personally prefer to not do this as we don't really need to run stable/beta/nightly on OSX,  could the previous iteration of `include:` be used instead with explicit additions if necessary?

Reply:
It seems simpler to just exclude beta and nightly on OS X with this matrix.

Reply:
BTW I'd still prefer to not do this, but if you'd like I can take care of switching it back.

Reply:
I've got it excluding beta and nightly below this, it's only running two OS X jobs.

Reply:
Ah yeah sure, I just prefer a bunch of explicit `include:` lines as historically I've found working with a matrix doesn't really work well in the long run (I don't mind cleaning this up to my own personal linking, which may be a bit odd, though)

Reply:
I think this is nicer than manually declaring 8 jobs but it's your repo. I can fix it if you prefer.